bundle/jsky.elevation.plot/src/main/java/jsky/plot/ElevationPlotMenuBar.java

```diff
@@ -71,7 +71,6 @@ protected JMenu createFileMenu() {
     protected JMenu createViewMenu() {
         JMenu menu = new JMenu(_I18N.getString("view"));
         menu.add(_plotPanel.getDateAction());
-        menu.add(_plotPanel.getTargetsAction());
         menu.add(_showLegendMenuItem = _createViewLegendMenuItem());
```

Comment:
The target list menu item is not added anymore, and thus doesn't need to be removed (see above).